ports/esp32/boards/BPI_LEAF_S3/mpconfigboard.h

```diff
@@ -0,0 +1,8 @@
+#define MICROPY_HW_BOARD_NAME               "BPI-Leaf-S3"
+#define MICROPY_HW_MCU_NAME                 "ESP32-S3"
+
+#define MICROPY_PY_BLUETOOTH                (0)
```

Comment:
Is there a reason to disable Bluetooth?

Reply:
Sorry for the delay. 

Just because the esp32s2 boards disable Bluetooth.
So I used the configurations of esp32s2 in the early stage.

I should correct some configurations.

Reply:
I made a force-pushed，make commits clean，and fix ‘Check commit message formatting ’ , now All checks have passed.